spark/dl/src/main/scala/com/intel/analytics/bigdl/dlframes/DLClassifier.scala

```diff
@@ -34,6 +34,8 @@ import scala.reflect.ClassTag
  * @param criterion  BigDL criterion method
  * @param featureSize The size (Tensor dimensions) of the feature data.
  */
+@deprecated("`DLClassifier` is deprecated." +
+  "com.intel.analytics.bigdl.dlframes is deprecated in BigDL 0.11", "0.10.0")
```

Comment:
"and will be removed in future releases"

Reply:
done